### ISSUE TITLE
Exception when rollbar isn't initialized

### ISSUE THREAD

**Issue by nemo** [rollbar/node_rollbar#60]
Hey folks,

Was writing some tests and it looks like rollbar throws an exception when it isn't properly initialized. 

Seems like this is a risky move and even if it's intentional, it should be throwing a better exception that the one below.

Stacktrace:
```
     Uncaught TypeError: Cannot read property 'postItem' of undefined
      at node_modules/rollbar/lib/notifier.js:301:16
      at node_modules/rollbar/lib/notifier.js:285:5
      at node_modules/rollbar/node_modules/async/lib/async.js:708:13
      at node_modules/rollbar/node_modules/async/lib/async.js:49:16
      at node_modules/rollbar/node_modules/async/lib/async.js:263:32
      at node_modules/rollbar/node_modules/async/lib/async.js:40:16
      at node_modules/rollbar/node_modules/async/lib/async.js:705:17
      at addReqData (node_modules/rollbar/lib/notifier.js:272:5)
      at node_modules/rollbar/node_modules/async/lib/async.js:699:13
      at iterate (node_modules/rollbar/node_modules/async/lib/async.js:256:13)
      at node_modules/rollbar/node_modules/async/lib/async.js:268:29
      at node_modules/rollbar/node_modules/async/lib/async.js:40:16
      at node_modules/rollbar/node_modules/async/lib/async.js:705:17
      at node_modules/rollbar/lib/notifier.js:107:5
      at node_modules/rollbar/lib/parser.js:288:12
      at node_modules/rollbar/lib/parser.js:243:11
      at node_modules/rollbar/node_modules/async/lib/async.js:49:16
      at node_modules/rollbar/node_modules/async/lib/async.js:263:32
      at node_modules/rollbar/node_modules/async/lib/async.js:40:16
      at gatherContextLines (node_modules/rollbar/lib/parser.js:231:7)
      at node_modules/rollbar/node_modules/async/lib/async.js:162:20
      at iterate (node_modules/rollbar/node_modules/async/lib/async.js:256:13)```

**Comment by elliot-a** [comment]
I've just had the same issue on a test server - for some reason roll bar didn't initialise and I now have literally millions of lines of [TypeError: Cannot read property 'postItem' of undefined] in my logs.

**Comment by coryvirok** [comment]
Thanks for the bug report! This should be fixed now in v0.6.1.
